File: src/main/java/damian/tab/websocket/model/Player.java
package damian.tab.websocket.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;
import org.springframework.web.socket.WebSocketSession;

@AllArgsConstructor
@RequiredArgsConstructor
@Data
public class Player {
    private short x;
    private short y;
    private short score;
    private final String nick;
    private final WebSocketSession session;
}

File: src/frontend/src/app/game/player.ts
export class Player{
  nick: string;
  x: number;
  y: number;
  score: number;
  color: string;

  constructor(nick: string, x: number, y: number, score: number, color: string) {
    this.nick = nick;
    this.x = x;
    this.y = y;
    this.score = score;
    this.color = color;
  }
}
